### ISSUE TITLE
Some tools can't be put on the Tool Rack

### ISSUE THREAD

**Issue by NinaVictoria** [Nuchaz/BiblioCraft#646]
I found I can not put the pickaxe and shovel (from Tinkers' Construct 2 ) on this Tool Rack,it is usless after I revised the config.And other tools are able to be put on the Tool Rack.

**Comment by GolDmiAna** [comment]
This error is on version TConstruct-1.12.2-2.12.0.115. In version
TConstruct-1.12.2-2.12.0.113 does not yet exist. On version TConstruct-1.12.2-2.12.0.123 it is no more.

**Comment by NinaVictoria** [comment]
> 此错误发生在TConstruct-1.12.2-2.12.0.115版本上。在
> TConstruct-1.12.2-2.12.0.113 版本中尚不存在。在TConstruct-1.12.2-2.12.0.123版本上它已不复存在。

Well, I owned version TConstruct-1.10.2-2.6.5,and BiblioCraft[v2.2.5][MC1.10.2]

**Comment by GolDmiAna** [comment]
Is your default language English?

**Comment by NinaVictoria** [comment]
> 你的默认语言是英语吗？

 No,default language is Chinese,but I think it's not about language but about others.

**Comment by GolDmiAna** [comment]
My default language was also not English. I switched to this — it all worked.

**Comment by NinaVictoria** [comment]
> 我的默认语言也不是英语。我切换到了 - 这一切都奏效了。

well,I tried it and it worked.Then I add all the key word to the config in Chinese,problem solved.
Thank you for your help.
